From 38ff9dfbe5d134347638e61bd265065a0c97c0c5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Thu, 2 Nov 2017 17:04:49 +0100
Subject: [PATCH] dmsquash-generator.sh: properly escape backslash in path

We need an actual bashslash in the filename on disk.

https://bugzilla.redhat.com/show_bug.cgi?id=1508794
---
 modules.d/90dmsquash-live/dmsquash-generator.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/90dmsquash-live/dmsquash-generator.sh b/modules.d/90dmsquash-live/dmsquash-generator.sh
index d47cf14a..fe190012 100755
--- a/modules.d/90dmsquash-live/dmsquash-generator.sh
+++ b/modules.d/90dmsquash-live/dmsquash-generator.sh
@@ -70,7 +70,7 @@ ROOTFLAGS="$(getarg rootflags)"
     else
         echo "What=/dev/mapper/live-rw"
         [ -n "$ROOTFLAGS" ] && echo "Options=${ROOTFLAGS}"
-        _dev=dev-mapper-live\x2drw
+        _dev=$'dev-mapper-live\\x2drw'
     fi
 } > "$GENERATOR_DIR"/sysroot.mount
 
